saleor/plugins/vatlayer/plugin.py

```diff
@@ -48,8 +49,39 @@ class VatlayerPlugin(BasePlugin):
     PLUGIN_NAME = "Vatlayer"
     META_CODE_KEY = "vatlayer.code"
     META_DESCRIPTION_KEY = "vatlayer.description"
-    DEFAULT_CONFIGURATION = [{"name": "Access key", "value": None}]
+
+    DEFAULT_CONFIGURATION = [
+        {"name": "Access key", "value": None},
+        {"name": "source_country", "value": None},
+        {"name": "countries_to_calculate_taxes_from_source", "value": None},
+        {"name": "excluded_countries", "value": None},
+    ]
+
     CONFIG_STRUCTURE = {
+        "source_country": {
```

Comment:
I think in the VAT-terminology, this is called "origin country" ([source](https://www.europarl.europa.eu/RegData/etudes/BRIE/2017/603265/EPRS_BRI(2017)603265_EN.pdf), see the "Summary" section). I would rename it here and in the labels and help texts.